packages/git/src/browser/diff/git-diff-widget.tsx

```diff
@@ -232,7 +238,15 @@ export class GitDiffWidget extends GitNavigableListWidget<GitFileChangeNode> imp
             ref={ref => this.listView = ref || undefined}
             id={this.scrollContainer}
             files={files}
-            addDiffListKeyListeners={this.addGitDiffListKeyListeners} />;
+            addDiffListKeyListeners={this.addGitDiffListKeyListeners}
+            onListContainerReady={this.onListContainerReady} />;
+    }
+
+    protected onListContainerReady = (c?:HTMLDivElement) => this.doOnListContainerReady(c);
+    protected doOnListContainerReady(c?:HTMLDivElement) {
```

Comment:
missing returns-typedef

Reply:
What is `c`? see https://github.com/theia-ide/theia/wiki/Coding-Guidelines#whole-words-names

Reply:
Sorry for that. Solved both.